src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -153,11 +164,15 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
               </EuiText>
             )}
             {formBody}
-            {this.renderCopyOnSave()}
           </EuiForm>
         </EuiModalBody>
 
-        <EuiModalFooter>
+        <EuiModalFooter
+          css={css`
+            align-items: center;
+          `}
+        >
+          <EuiFlexItem grow>{this.renderCopyOnSave()}</EuiFlexItem>
```

Comment:
Two notes here:

1. Similar to a previous comment, I tend to not recommend using an `EuiFlexItem` without a parent `EuiFlexGroup` (and vice versa). In this case, I'd recommend adding an `EuiFlexGroup` with two `EuiFlexItem` children (one for the switch, the other for the cancel and save buttons).
2. This `EuiFlexItem` should be included as part of the conditional rendering of the switch, as it currently causes the `EuiFlexItem` to always render, regardless if the switch is present or not.

Reply:
Addressed in https://github.com/elastic/kibana/pull/159057